Formula/ffmpeg.rb

```diff
@@ -95,6 +103,15 @@ def install
       --host-ldflags=#{ENV.ldflags}
     ]
 
+    if build.with? "openh264"
+      resource("openh264-1.5.0").stage do
+        system "make", "install-shared", "PREFIX=#{libexec}/openh264-1.5.0"
+        chmod 0444, "#{libexec}/openh264-1.5.0/lib/libopenh264.dylib"
```

Comment:
I had copy-pasted from here https://github.com/Homebrew/homebrew-core/blob/master/Formula/openh264.rb#L19 and then fixed up the prefixes. But this was a gross oversight, I admit :)